WATOLINK/bci-gaming
# Created Menu template

## ClickUp ticket link

[Game - Menu Interface](https://app.clickup.com/t/860pxkur1)



## Implementation description
- Created a launch screen which is disabled for now 
- Created main menu with play button (with a script to go to next scene when pressed), and bci, info, and settings menu ( which open a blank modal when pressed) 



## Steps to test
1. Play the game and ensure you can see the main menu with no modals open by default
2. Click the bci menu, info and settings buttons and ensure that a blank modal open
3. Press play and the game should exit/ throw an error that the next scene doesn't exist yet. 



## What should reviewers focus on?
This is just a skeleton so we can change all the styling to reflect our game design.


## Checklist
